src/cjs-download/downloadCjsFile.ts

```diff
@@ -1,7 +1,8 @@
 import axios from "axios"
 
-export default (downloadLocation: string): Promise<Buffer> => axios({
-  url: downloadLocation,
-  method: "GET",
-  responseType: "arraybuffer",
-}).then((response) => Buffer.from(response.data))
+export default (downloadLocation: string): Promise<Buffer> =>
```

Comment:
Should this be an async function?

Reply:
No. It's not using await

Reply:
It's returning a promise so it doesn't need to be async